networkx/drawing/layout.py

```diff
@@ -983,6 +983,14 @@ def spiral_layout(G, scale=1, center=None, dim=2, resolution=0.35, equidistant=F
     --------
     >>> G = nx.path_graph(4)
     >>> pos = nx.spiral_layout(G)
+    >>> nx.draw(G, pos=pos)
+    
+    References
+    ----------
+    .. [1] Se-Hang Cheong, Yain-Whar Si
+       "Force-directed algorithms for schematic drawings and placement: A survey",
+       [2019]
+       https://journals.sagepub.com/doi/full/10.1177/1473871618821740
```

Comment:
```suggestion
    .. [1] Cheong, Se-Hang, and Yain-Whar Si. "Force-directed algorithms for schematic drawings and
        placement: A survey." Information Visualization 19, no. 1 (2020): 65-91.
        https://doi.org/10.1177%2F1473871618821740
```

I personally prefer directly using DOI links wherever possible :)

Reply:
okay sir, I will look more into DOI links, thanks.